.github/workflows/R-linux.yml

```diff
@@ -0,0 +1,133 @@
+      desc_unchanged: ${{ steps.cache-desc.outputs.cache-hit }}
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'linux nocache')"
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - id: cache-desc
+        uses: actions/cache@v2
+        with:
+          path: |
+            ./DESCRIPTION
+          key: check-desc-${{ hashFiles('DESCRIPTION') }}
+
+      - id: cache-dirs
+        uses: actions/cache@v2
+        with:
+          path: |
+            ./R
+            ./tests
+            ./vignettes
+          key: check-files-${{ hashFiles('R/**') }}-${{ hashFiles('vignettes/**') }} -${{ hashFiles('tests/**') }}
+
+  deps:
+    runs-on: ubuntu-latest
+    needs: setup
+    if: needs.setup.outputs.desc_unchanged != 'true'
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: release
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::rcmdcheck
+          needs: check
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::covr
+          needs: coverage
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: local::.
+
+  check:
+    runs-on: ubuntu-latest
+    needs: [setup]
+    if: "!contains(github.event.head_commit.message, 'linux nocheck')"
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: release
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::rcmdcheck
+          needs: check
+
+      - uses: r-lib/actions/check-r-package@v2
+        with:
+          upload-snapshots: true
+
+  covrpage:
+    needs: [check]
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'linux nocovrpage')"
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: release
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::covr
+          needs: coverage
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: local::.
+
+      - name: covrpage-setup
+        run: pak::pkg_install(c('hadley/emo','yonicd/covrpage@actions'), ask = FALSE)
+        shell: Rscript {0}
+
+      - name: covrpage
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Actions"
+          Rscript -e 'remotes::install_github("hadley/emo")' \
+                  -e 'remotes::install_github("yonicd/covrpage@actions")' \
```

Comment:
do we need that specific branch? Did you already try to make a PR towards the main `covrpage` repo? just thinking for longer term maintenance that would be better.

Reply:
Yes. The links for the tests are taking you directly to the lines that were not run. There are icons next to tests that gave either warnings/failures as a way to know which ones were the offending ones. 

I merge that branch into main, so that can be amended to be `yonicd/covrpage`.